libs/wallet/src/web3-react/updater.ts

```diff
@@ -86,19 +90,39 @@ function _useSafeInfo(walletInfo: WalletInfo): GnosisSafeInfo | undefined {
       }
     }
 
-    const interval = setInterval(update, SAFE_INFO_UPDATE_INTERVAL)
+    const updateSafeAppsSdkInfo = async () => {
```

Comment:
Since this should only update `isReadOnly`, I suggest to make the name more explicit:

```suggestion
    const updateIsReadOnlyFromSafeAppsSdkInfo = async () => {
```